src/legacy/core_plugins/tagcloud/public/tag_cloud_fn.ts

```diff
@@ -63,25 +60,26 @@ export const tagcloud = () => ({
     metric: {
       types: ['vis_dimension'],
       help: i18n.translate('tagCloud.function.metric.help', {
-        defaultMessage: 'metric dimension configuration'
+        defaultMessage: 'metric dimension configuration',
       }),
       required: true,
     },
     bucket: {
       types: ['vis_dimension'],
       help: i18n.translate('tagCloud.function.bucket.help', {
-        defaultMessage: 'bucket dimension configuration'
+        defaultMessage: 'bucket dimension configuration',
       }),
     },
   },
-  fn(context, args) {
+  fn(context: any, args: any) {
     const visConfig = {
       scale: args.scale,
       orientation: args.orientation,
       minFontSize: args.minFontSize,
       maxFontSize: args.maxFontSize,
       showLabel: args.showLabel,
       metric: args.metric,
+      bucket: undefined,
```

Comment:
I don't think this needs to be added here. I think we only want to add the `bucket` key to `visConfig` if it has a value (lines 85-87)